apps/website/src/pages/docs/datepicker.mdx

```diff
@@ -14,44 +14,74 @@ import * as AllExamples from '~/examples';
 
-### Inline
+### Localized
 
-The inline date picker was styled to be used in forms. The horizontally centered label helps the component belong in a form without breaking the flow.
+You are able to create a localized version of the date picker that uses local names for week days and months.
 
-### With time
+<LiveExample src='DatePicker.localized.tsx' truncate={true}>
+  <AllExamples.DatePickerLocalizedExample client:load />
+</LiveExample>
 
-If you need to include the time, there are multiple ways to do so. If it is only required to select a time within 15 minute increments, it is suggested to combine the time picker with the date picker.
+### Date + time selection
 
-You may choose to display the time in military format and exclude the period picker.
+If you need to include the time, there are multiple ways to do so. Below is the default variant, which uses twenty-four hour time.
 
-If the user needs the ability to enter a specific time, we suggest displaying the time within it’s own select menu within close proximity of the date picker. The user can select a time from the dropdown menu or manually enter a specific time.
+<LiveExample src='DatePicker.withtime.tsx' truncate={true}>
+  <AllExamples.DatePickerWithTimeExample client:load />
+</LiveExample>
 
-### Date spans
+And here is the variant that uses twelve-hour time with meridiem:
 
-There may be some cases where you need to display a span of dates. When this occurs, the selection box spans the entirety of the date span.
+<LiveExample src='DatePicker.withcombinedtime.tsx' truncate={true}>
+  <AllExamples.DatePickerWithCombinedTimeExample client:load />
+</LiveExample>
+
+### Date range
+
+There may be some cases where you need to display a range of dates. When this occurs, the selection box spans the entirety of the date range.
+
+<LiveExample src='DatePicker.daterange.tsx' truncate={true}>
+  <AllExamples.DatePickerDateRangeExample client:load />
+</LiveExample>
+
+### Some dates disabled
+
+You might also need to have certain dates disabled, in case the range of dates that can be selected for your purpose is limited.
+
+<LiveExample src='DatePicker.datesdisabled.tsx' truncate={true}>
+  <AllExamples.DatePickerDatesDisabledExample client:load />
+</LiveExample>
 
 ### Year skipping
 
 To improve user experience, date pickers support year skipping buttons. Clicking on the double chevron icon will move the calendar a full year forward or back.
 
+<LiveExample src='DatePicker.withyear.tsx' truncate={true}>
+  <AllExamples.DatePickerWithYearExample client:load />
+</LiveExample>
+
+## Usage
+
+A DatePicker can be featured on your page in multiple ways. One can be added directly to your page so that it is visible as soon as the page loads. Additionally, a DatePicker can be added to a menu button on your page so that the user can toggle its visibility.
```

Comment:
```suggestion
A date picker can be featured on your page in multiple ways. One can be added directly to your page so that it is visible as soon as the page loads. Additionally, a date picker can be added to a menu button on your page so that the user can toggle its visibility.
```

Reply:
done